hack/lint-dependencies.sh

```diff
@@ -25,6 +25,9 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
+kube::golang::setup_env
+make -C "${KUBE_ROOT}" WHAT=cmd/dependencyverifier
```

Comment:
can drop this, we're running directly below

Reply:
Removed.